utils/hashPassword.js

```diff
@@ -14,11 +14,11 @@ const crypto = require('crypto')
 // increase this over time. Existing passwords can be refreshed on login by
 // calling hashPassword again.
 //
-// 200 iterations is chosen to take about 5 ms on hardware you might commonly
+// 1024 iterations is chosen to take about 2-5 ms on hardware you might commonly
 // find in a server deployment. Users who need more security or more performance
 // are encouraged to pass a different value via the second parameter in
 // hashPassword.
-const PBKDF2_ITERATIONS = 256
+const PBKDF2_ITERATIONS = 1024
```

Comment:
Good point. If we switch in the future to an API key, we should remember to increase the iteration count.

Did you see the other points I mentioned in the summary? Specifically in line [88](https://github.com/interledgerjs/five-bells-shared/pull/166/files/a83af52d977680d7b07bb32e7e9850e1e6782068#diff-1b17a138d1dfd139e3ad85fd1d354d0bR88), `PBKDF2_ITERATIONS` should be replaced by `iterations`.